public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico?" type="image/x-icon" />
```

Comment:
```suggestion
    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" type="image/x-icon" />
```
ich denke das fragezeichen braucht es nicht

Reply:
Braucht es, sonst wird das ICON nicht angezeigt.